sympy/series/formal.py

```diff
@@ -972,8 +978,19 @@ def infinite(self):
 
         return self.ind + inf_sum
 
+    @property
+    def fps_sym(self):
+        """Returns whether the function contains symbolic terms"""
+        ind = self.ind
+        s = ind.free_symbols.difference({self.x})
+        if len(s) == S.One:
+            return True
+        return False
```

Comment:
Do we need this? I think having just a ``free_symbols`` property for ``FormalPowerSeries`` should suffice.